layouts/_default/baseof.html

```diff
@@ -24,9 +24,7 @@
       <link rel="canonical" href="{{ .Permalink }}">
     {{ end }}
 
-    {{ $fa := resources.Get "fonts/forkawesome-webfont.woff2" }}
-    {{ $faFingerprint := $fa | fingerprint }}
-    <link rel="preload" href="{{ $fa.RelPermalink }}?v=1.1.7" as="font" type="font/woff2" integrity="{{ $faFingerprint.Data.Integrity }}" crossorigin>
+    <link rel="preload" href="/fonts/forkawesome-webfont.woff2?v=1.1.7" as="font" type="font/woff2" crossorigin>
```

Comment:
The same param is used inside css. Removing it here will cause browser to download woff again, regardless of this preload. At least this is what happened to me when trying to remove it in #520 .

Reply:
This param was always here and same one is used inside the forkawesome css. I tried to remove it from here in #520 , but it caused the browser to download woff again when css was loaded. We would need to remove it here and in css, but IMHO it's better to leave both as is.

Reply:
This param was always here and same one is used inside the forkawesome css. I tried to remove it from here when working on #520 , but it caused the browser to download woff again when the aforementioned css was loaded. We would need to remove it here and in css, but IMHO it's better to leave both as is.

Reply:
This param was always here and the same one is used inside the forkawesome css. I tried to remove it from here when working on #520 , but it caused the browser to download woff again when the aforementioned css was loaded. We would need to remove it here and in css, but IMHO it's better to leave both as is.

Reply:
This param was always here and same one is used inside the forkawesome css. I tried to remove it from here when working on #520 , but it caused my browser to download woff2 again when the aforementioned css was loaded. We would need to remove it here and in css, but IMHO it's better to leave both as is.

Reply:
This param was always here and same one is used inside the forkawesome css. I tried to remove it from here when working on #520 , but it caused my browser to download woff2 again when the aforementioned css was loaded. We would need to remove it from here and from css, but IMHO it's better to leave both as is.

Reply:
This param has always been here. I tried to remove it when working on #520 , but it caused my browser to download woff2 again when the forkawesome css was loaded. I think it's because the css uses this param too. We would need to remove it from here and from the css, but IMHO it's better to leave both as is.

Reply:
Ok. I'm going to test it later.